src/page-modules/contact/means-of-transport/means-of-transport-form-machine.ts

```diff
@@ -77,13 +79,31 @@ export const meansOfTransportFormMachine = setup({
         } else {
           context.errorMessages[inputName] = [];
         }
+
         return {
           ...context,
           [inputName]: value,
         };
       }
       return context;
     }),
+    onTransportModeChange: assign(({ context, event }) => {
+      if (event.type === 'ON_TRANSPORTMODE_CHANGE') {
+        const { inputName, value } = event;
+        if (inputName === 'transportMode')
```

Comment:
`onTransportModeChange` should only be used by inputs handling transportMode. You shouldn't need to check the input name here.